app/src/main/java/com/github/mik629/android/fundamentals/data/network/model/ActorDTO.kt

```diff
@@ -1,12 +1,16 @@
 package com.github.mik629.android.fundamentals.data.network.model
 
+import com.github.mik629.android.fundamentals.domain.model.Actor
 import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
 
 @JsonClass(generateAdapter = true)
-data class Actor(
+data class ActorDTO(
     val id: Int,
     val name: String,
     @Json(name = "profile_path")
     val avaPath: String?
-)
\ No newline at end of file
+) {
+    fun toActor() =
```

Comment:
extensions with named params

Reply:
why extension? what's wrong with simple method?

Reply:
just to use same approach for all cases
also this is dto -> mapping function is not part a of business logic